docs/section11/Rule11-13.md

```diff
@@ -0,0 +1,49 @@
+    - look at each building segment in the building: `for building_segment in building.building_segments:`
+        - get the service water heating uses in the building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(P_RMD, building_segment.id)`
+        - look at each service water heating use id: `for swh_use_id in service_water_heating_use_ids:`
+            - get the swh_use using get_component_by_ID: `swh_use = get_component_by_ID(P_RMD, swh_use_id)`
+            - check to see if the use has SWH loads: `if swh_use.use > 0:`
+                - the rule is applicable: `is_applicable = true`
+                - now get the energy required to heat this swh use using the function get_energy_required_to_heat_swh_use: `energy_required_by_space_id = get_energy_required_to_heat_swh_use(swh_use, B_RMD, building_segment)`
+                - if any of the energy required is UNDETERMINED, we need to make BTU_PER_YEAR UNDETERMINED: `if "UNDETERMINED" in energy_required_by_space_id.values():`
+                    - set BTU_PER_YEAR to UNDETERMINED: ` service_water_heating_info["BTU_PER_YEAR"] = "UNDETERMINED"`
+                    - set BTU_PER_SF_PER_YEAR to UNDETERMINED: ` service_water_heating_info["BTU_PER_SF_PER_YEAR"] = "UNDETERMINED"`
+                - otherwise, if BTU_PER_YEAR is not UNDETERMINED, add the sum of the values of energy_required_by_space_id to the total: `if service_water_heating_info["BTU_PER_SF_PER_YEAR"] != "UNDETERMINED":`
+                    - add the sum of the values of to the total: ` service_water_heating_info["BTU_PER_YEAR"] += sum(energy_required_by_space_id.values())`
+        - if the BTU_PER_YEAR is not UNDETERMINED, we now calculate the total building area to give BTU_PER_SF_PER_YEAR: `if service_water_heating_info["BTU_PER_YEAR"] != "UNDETERMINED":`
+            - create a variable floor_area (ft2): `floor_area = 0`
+            - look at each space in the building: `for space in Building.... spaces:`
+                - add the space floor area to the floor_area: `floor_area += space.floor_area`
+            - check if the floor area is greater than 0: `if floor_area > 0:`
+                - calculate the BTU_PER_SF_PER_YEAR: `service_water_heating_info["BTU_PER_SF_PER_YEAR"] = service_water_heating_info["BTU_PER_YEAR"] / floor_area`
+      
+    
+    ## Rule Assertion:
+    - if the boolean is_applicable is equal to true, the rule is applicable, return UNDETERMINED and a note letting the reviewer know what to check: `if is_applicable: UNDETERMINED raise_message: "This building has service water heating loads.  Confirm that service water heating energy consumption is calculated explicitly based upon the volume of service water heating required and the entering makeup water and leaving service water heating temperatures.  Entering water temperatures shall be estimated based upon the location. Leaving temperatures shall be based upon the end-use requirements."`
```

Comment:
Does the calculated BTU per square foot per year need to be included in the output message?

Reply:
maybe a note to clarify that the intent is for the data in `service_water_heating_info` to end up in the calc_vals that gets printed along with the message